plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -108,7 +108,9 @@ export default class NodesTable extends React.Component<
       case "zone":
         return compareByZone;
       case "health":
-        return UnitHealthUtil.getHealthSortFunction;
```

Comment:
is anything lost by bypassing this function?

Reply:
no. we know exactly by what we want to have our data sorted. and we reach for exactly that function.

also note that `getHealthSortFunction` does not do its job on master.